### PR TITLE
added wgms3d recipe

### PR DESCRIPTION
**Pull request by joamatab**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [ x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/wgms3d/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  build:
+    - blas 1.1 {{ variant }}
+    - openblas 0.2.19|0.2.19.*
+    - cmake
+    - gcc  # [not win]
+  run:
+    - blas 1.1 {{ variant }}
+    - openblas 0.2.19|0.2.19.*
+    - libgcc  # [linux]
+    - libgfortran  # [osx]
```
Comment (davidbrochart) [inline recipes/wgms3d/meta.yaml:60]: Did you forget to delete these requirements? You have other ones below.

File: recipes/wgms3d/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  # sha256: d58439c548433adcda98e695be53e526ba940a4b9c44fb9a05d92cd495cdd47f
+  # sha256 is the preferred checksum -- you can get it for a file with:
+  #  `openssl sha256 <file name>`.
+  # You may need the openssl package, available on conda-forge:
+  #  `conda install openssl -c conda-forge``
+
+source:
+    fn: wgms3d-{{ version }}.tar.gz
+    url: http://www.soundtracker.org/raw/wgms3d/wgms3d-{{ version }}.tar.gz
+    sha256: c68b1b868d84d3351622ec5b4df5b6af1e0ac916
```
Comment (davidbrochart) [inline recipes/wgms3d/meta.yaml:30]: It seems the SHA is `cba974d049db4162904602858bb0bc2069af8c83c9360e2c9eadb1ef560ac3fa`.

File: recipes/wgms3d/meta.yaml (hunk 1 above)
Comment (joamatab) [reply recipes/wgms3d/meta.yaml]: Thank you David, I removed the other requirements

File: recipes/wgms3d/meta.yaml (hunk 2 above)
Comment (joamatab) [reply recipes/wgms3d/meta.yaml]: Thank you David

File: recipes/wgms3d/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+  # See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python for more details.
+  # noarch: python
+  # number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  # script: "{{ PYTHON }} -m pip install . -vv"
+
+build:
+  number: 200
```
[the remark this answers is not in the document]
Comment (joamatab) [reply recipes/wgms3d/meta.yaml]: thank you David

**Comment by joamatab** [comment]
Im getting the error

configure: error: Need Boost C++ library headers.

How could I add boost C++ library headers in the yaml? (I think I already did)

File: recipes/wgms3d/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - cmake
+    - make
+    - arpack
+    - superlu
+    - boost
+    - boost-cpp
```
Comment (davidbrochart) [inline recipes/wgms3d/meta.yaml:32]: I think `boost` and `boost-cpp` should be in the host requirements instead.

File: recipes/wgms3d/build.sh [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+# MY_LDFLAGS="-L${LIB} ${RPATH_FLAGS}"
+# MY_CPPFLAGS="-I${PREFIX}/include"
+# BLAS="${LIB}/libopenblas${SO_EXT}"
+# CC="${BIN}/gcc"
+# CPP="${BIN}/g++"
+# FOR="${BIN}/gfortran"
+
+# ./configure --prefix="${PREFIX}" F77="${FOR}" CC="${CC}" CXX="${CPP}" LDFLAGS="${MY_LDFLAGS}" CPPFLAGS="${MY_CPPFLAGS}" --with-blas="${BLAS}"
+# ./configure --prefix=${PREFIX}  --with-arpack --with-superlu
+./configure  --with-arpack --with-superlu
```
Comment (davidbrochart) [inline recipes/wgms3d/build.sh:26]: I think you should specify the prefix:
```suggestion
./configure --prefix=${PREFIX} --with-arpack --with-superlu
```

**Pull request closed without merging**